partials/language-specific-deploy/php.md

```diff
@@ -196,13 +196,27 @@ In order to prevent this behavior, you can add the following statements in a `.h
 DirectorySlash Off
 RewriteEngine On
 RewriteCond %{REQUEST_FILENAME} -d
-RewriteRule ^(.+[^/])$          %{HTTP:X-Forwarded-Proto}://%{HTTP_HOST}/$1/  [R=301,L,QSA]
+RewriteRule ^(.+[^/])$ %{HTTP:X-Forwarded-Proto}://%{HTTP_HOST}/$1/ [R=301,L,QSA]
 ```
 
 These statements will keep the former protocol of the request when issuying the redirect. Assuming that the header X-Forwarded-Proto is always filled (which is the case on our platform).
 
 If you want to force all redirects to HTTPS, you can replace `%{HTTP:X-Forwarded-Proto}` with `https`.
 
+### Change FastCGI module
+
+You can choose between two FastCGI modules, `fastcgi` and `proxy_fcgi`.
+
+To choose between this two modules you must use `CC_CGI_IMPLEMENTATION` environment variable with `fastcgi` or `proxy_fcgi` as value.
+
+{{ < alert "warning" > }}
+We recommend to use `proxy_fcgi` over the `fastcgi` implementation because it is not maintained anymore but to prevent unexpected behaviors with historical applications it has been kept as the default one.
+{{ < /alert > }}
+
+If you have issues with content downloading, it could be related to `fastcgi` module which is not working correctly in combination with `deflate` module as the `Content-Length` header is not updated to the new size of the encoded content.
```

Comment:
```suggestion
If you have issues with downloading content, it could be related to the `fastcgi` module not working correctly in combination with the `deflate` module, as the `Content-Length` header is not updated to the new size of the encoded content.
```